test-requirements.txt

```diff
@@ -15,3 +15,4 @@ py>=1.5.2
 virtualenv<20
 setuptools
 importlib-metadata==0.20
+Cython
```

Comment:
I'd rather not add Cython as a dependency. Instead, it would be better to have the test automatically skipped if Cython is not installed.